cmd/kubeadm/app/constants/constants.go

```diff
@@ -199,6 +199,20 @@ const (
 	// We need at least ten, because the DNS service is always at the tenth cluster clusterIP
 	MinimumAddressesInServiceSubnet = 10
 
+	// MaximumBitsForServiceSubnet defines maximum possible size of the service subnet in terms of bits.
+	// For example, if value is 20, then the largest supported service subnet is /12 for IPv4 and /108 for IPv6.
+	// Note however that anything in between /108 and /112 will be clamped to /112 due to the limitations of the underlying allocation logic.
```

Comment:
did we had a ticket for that in k/k that we can link to from here?
maybe have a `TODO: ...` to update this once/if the bitmap problems are resolved?

Reply:
added the kep that explains the whole thing